src/commands/importIdentifier.ts

```diff
@@ -0,0 +1,108 @@
+import * as vscode from 'vscode';
+import { CommandNames } from './constants';
+import * as request from 'request-promise-native';
+import * as cheerio from 'cheerio';
+import * as LRU from 'lru-cache';
+import * as _ from 'lodash';
+import * as yaml from 'js-yaml';
+
+const askHoogle = async (variable: string): Promise<any> => {
+  return await request({
+    url: `https://hoogle.haskell.org/?hoogle=${variable}&scope=set%3Astackage&mode=json`,
```

Comment:
I think it would be better to ask HIE than `hoogle.haskell.org` because HIE already supports and uses local hoogle database...

Reply:
Yeah, it could query the local Hoogle DB before falling back to hoogle.haskell.org

Reply:
And ideally the actual functionality should move into hie itself, so it is available for other IDE environments too.  But one step at a time.